### ISSUE TITLE
Remove GDX Facebook from Third-party list

### ISSUE THREAD

**Issue by JojoIce** [tommyettinger/gdx-liftoff#43]
It sadly looks like GDX Facebook has died. Hasn't been updated in 2 years and it stopped working quite some time ago at least for iOS. It will actually block your app from being added to app store and not with a very clear hint that it is the GDX Facebook that is the culprit.

_Originally posted by @TomGrill in https://github.com/TomGrill/gdx-facebook/issues/63#issuecomment-708965353_

If anybody had suggestions for how to replace this with something else I will be really happy, since that is what I used until it started blocking my apps.

Having this in the 3rd party list will probably only do bad at this point.

Alternatively update the hover text to explain the problems. Maybe it somewhat works for android but very outdated.

**Comment by tommyettinger** [comment]
Yeah, I hope there's a fork or some replacement available. I know Facebook still has users, but they're mostly my parents' generation or in a different (non-US) country, I think. My parents' generation seems like a good target for microtransaction-based game monetization, though... So Facebook support might be a big win. I'll stop gdx-facebook from being added to iOS projects (because it blocks you from the App Store), and add some note to the hover text.

**Comment by JojoIce** [comment]
Well, it is actually quite useful for tracking and advertisement. I have made a testrun with one of my games at Voodoo, and they require games to have a facebook connection, so it is quite a big loss if you are want to do that. A testrun with Voodoo will mean that they test advertising it so you will get some traffic for free to your game and quite possibly they also give you feedback on things they think will improve the stats they gather. Also if Voodoo picks up your game you will have good chance of making good money. I will poke around or ask on Discord if anyone knows the way forward maybe.

**Comment by JojoIce** [comment]
I guess this can be closed since the commit is there. Thanks!
